rest-jersey/src/test/java/ws/ament/hammock/rest/jersey/RestApp.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 Hammock and its contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ws.ament.hammock.rest.jersey;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.core.Application;
+
+@ApplicationScoped
+public class RestApp {//extends Application { //??? Tested archive contains both Applications on classpath
```

Comment:
The arquillian extension automatically adds ws.ament.hammock.** to the archive, to help simplify usage.